misc/ballerinalang-data-mapper/src/test/resources/codeaction/source/dataMapper1.bal

```diff
@@ -0,0 +1,28 @@
+type Grades record {|
```

Comment:
Please add license header for all .bal files as well. Check all the places.

Reply:
Added license headers

Reply:
Added to all the .bal files